cumulusci/core/sfdx.py

```diff
@@ -65,6 +66,30 @@ def sfdx(
     return p
 
 
+def plog_command_options(options: T.List[str]) -> None:
```

Comment:
How about using `args` or `arguments` instead of options to maintain consistency with the `args` parameter above?